commands/invoke_test.go

```diff
@@ -23,14 +23,14 @@ func Test_invoke(t *testing.T) {
 	s := test.MockHttpServer(t, []test.Request{
 		{
 			Method:             http.MethodPost,
-			Uri:                "/function/" + funcName,
+			Uri:                "/function/" + funcName + ".openfaas-fn",
```

Comment:
Why is the namespace needed now?

It should still be possible to invoke without a namespace suffix, so I'm unsure why this change was made.

Reply:
The go-sdk which,  is now used to invoke functions, always needs a namespace to be specified. This is required for getting access tokens for functions if the function is authenticated (the full reference to a function, including the namespace, is required to request an access token)

This is why a function will always be invoked in the default namespace `openfaas-fn` if no namespace is provided explicitly.